.github/pull_request_template.md

```diff
@@ -1,8 +1,8 @@
-###  Summary
+### Summary
 
 Describe the goal of this pull request. Mention any related issue numbers from [slackapi/bolt-js/issues](https://github.com/slackapi/bolt-js/issues).
 
 ### Requirements (place an `x` in each `[ ]`)
 
 * [ ] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/.github/contributing.md) and have done my best effort to follow them.
-* [ ] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
\ No newline at end of file
+* [ ] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
```

Comment:
This isn't showing in the `diff` but a newline was added to the end of this file 👾